docs/permissions/plugin-authors/02-adding-a-basic-permission-check.md

```diff
@@ -146,7 +146,7 @@ In order to test the logic above, the integrators of your backstage instance nee
 -   async handle(): Promise<PolicyDecision> {
 +   async handle(
 +     request: PolicyQuery,
-+     user?: BackstageIdentityResponse,
++     _user?: BackstageIdentityResponse,
```

Comment:
By this point in the tutorial `user` isn't used, so if we don't `_` it, we'll get a tsc error.